app/views/pets/show.html.erb

```diff
@@ -24,9 +24,31 @@
   <strong>photo:</strong>
     <%= cl_image_tag @pet.photo, height: 300, width: 400, crop: :fill %>
 </p>
-<%# <p>
-  <strong><%= link_to 'Check Reviews', restaurant_reviews_path(@restaurant) </strong>
-</p> %>
+
+<h3>Reviews</h3>
+<% if @pet.reviews.empty? %>
+  <p>Be the first to let a review</p>
+<% else %>
+  <ul class="list-group">
+    <% @pet.reviews.each do |review| %>
+      <li class="list-group-item d-flex justify-content-between align-items-center">
+        <span class="badge badge-primary badge-pill">
+          <% counter = 0 %>
```

Comment:
why do you need a counter here if you use 5.times?

Reply:
To display stars